src/App.res

```diff
@@ -1,11 +1,35 @@
 %%raw("import './App.css'")
+
+@scope("JSON") @val
+external parseIntoMyData: string => array<TransactionType.t> = "parse"
+let initialTransaction: array<TransactionType.t> = [{text: "", amount: 0.}]
```

Comment:
Made it an empty array now there is also no need to filter out the blank transaction in the History Component. But in the last assignments PR review Bhavyam told me that : `Let's keep the structure of initial state something like this: [{text: "", amount: "0"}]`. So that's why I had done the same over here. As of now I had made the initalTransactionArray an empty one.